spec.txt

```diff
@@ -9188,6 +9197,28 @@ foo <a href="\*">
 ````````````````````````````````
 
 
+A block quote can prevent a line from being parsed as inline HTML,
+even though line breaks are allowed in tags:
+
+```````````````````````````````` example
+<a
+>
+.
+<p>&lt;a</p>
+<blockquote>
+</blockquote>
+````````````````````````````````
+
+
+```````````````````````````````` example
+<a
+b>
+.
+<p><a
+b></p>
+````````````````````````````````
+
```

Comment:
not sure this one is needed, it can never be a block quote. But not against it. Wonder what other folks think